aten/src/ATen/templates/TensorBody.h

```diff
@@ -319,6 +319,11 @@ class CAFFE2_API Tensor {
   /// Returns if a `Tensor` has quantized backend.
   bool is_quantized() const;
 
+  /// Returns true if a `Tensor` supports as_strided.
+  bool support_as_strided() const {
+    return impl_->support_as_strided();
```

Comment:
I probably wouldn't put this as an actual method on `Tensor`, since that implies its public API